modules/installation-creating-image-restricted.adoc

```diff
@@ -25,20 +36,26 @@ the {product-title} version that you install. Use the image versions that match
 your {product-title} version if they are available.
 ====
 
+ifdef::osp[]
 . Download the *{op-system-first} - OpenStack Image (QCOW)*.
+endif::osp[]
+ifdef::vsphere[]
+. Download the *{op-system-first} - vSphere*.
+endif::vsphere[]
 
 . Decompress the image.
 +
 [NOTE]
 ====
-You must decompress the {rh-openstack} image before the cluster can use it. The name of the downloaded file might not contain a compression extension, like `.gz` or `.tgz`. To find out if or how the file is compressed, in a command line, enter:
+You must decompress the image before the cluster can use it. The name of the downloaded file might not contain a compression extension, like `.gz` or `.tgz`. To find out if or how the file is compressed, in a command line, enter:
```

Comment:
For vsphere, the downloaded image is ended with "ova", which doesn't need to be decompressed although we run file command to get it is POSIX tar archive. Installer application will directly import it into vcenter